host/optee/linux/enclave.c

```diff
@@ -524,6 +569,10 @@ oe_result_t oe_terminate_enclave(oe_enclave_t* enclave)
     /* Finalize the context against OP-TEE */
     TEEC_FinalizeContext(&enclave->ctx);
 
+    /* Destroy the ecall id table */
+    if (enclave->ecall_id_table)
```

Comment:
Remove if check

Reply:
I think it's worth updating the our code base because we use different patterns in many places.

Reply:
I'm going to leave the check in since that is the majority coding pattern in use in the codebase.